lti/lti-common/src/java/org/sakaiproject/lti13/LineItemUtil.java

```diff
@@ -120,13 +121,16 @@ public static String constructExternalId(Map<String, Object> content, SakaiLineI
 	 */
 	// TODO: Remember to dream that someday this will be JSON :)
 	public static String constructExternalId(Long tool_id, Map<String, Object> content, SakaiLineItem lineItem)
+	{
+		Long content_id = (content == null) ? null : LTIUtil.toLongNull(content.get(LTIService.LTI_ID));
```

Comment:
I'm not sure where toLongNull is defined as I can't locate it in LTIUtil.java. Earlier this line used LTI13Util.getLongNull which had compelled the import of that class on line 58.